resources/client/cl_photo.ts

```diff
@@ -84,21 +85,29 @@ onNet(PhotoEvents.SEND_PHOTOS, (photos: string[]) => {
 
 function takePhoto() {
   // Return and log error if screenshot basic token not found
-  if (SCREENSHOT_BASIC_TOKEN === 'none') {
+  if (SCREENSHOT_BASIC_TOKEN === 'none' && config.images.url.includes('imgur')) {
     return console.error('Screenshot basic token not found. Please set in server.cfg');
   }
+
   exp['screenshot-basic'].requestScreenshotUpload(
-    'https://api.imgur.com/3/image',
-    'imgur',
+    config.images.url,
+    config.images.type,
     {
+      encoding: 'jpg',
       headers: {
         authorization: `Client-ID ${SCREENSHOT_BASIC_TOKEN}`,
```

Comment:
With the change to add custom image servers, there may be points where an authorization header isn't required. We should handle for that.